sdks/python/apache_beam/transforms/core.py

```diff
@@ -29,7 +29,7 @@
 from apache_beam.internal import util
```

Comment:
In Java the list of PTransforms with display data was much longer. Some components I didn't see here:

* Combine
* Window
* AvroIO
* XmlSource / XmlSink
* PipelineOptions